confignode/src/main/java/org/apache/iotdb/confignode/conf/ConfigNodeConfig.java

```diff
@@ -237,7 +239,9 @@ public class ConfigNodeConfig {
   private long configNodeSimpleConsensusLogSegmentSizeMax = 24 * 1024 * 1024L;
 
   /** RatisConsensus protocol, flow control window for ratis grpc log appender */
-  private long dataRegionRatisGrpcFlowControlWindow = 4 * 1024 * 1024L;
+  private long dataRegionRatisGrpcFlowControlWindow = 16 * 1024 * 1024L;
```

Comment:
Why change the default value?

Reply:
I think we should change the default value after a thorough evaluation.
Maybe we can leave this PR clean and separate the changes of default value to another PR. We may need that PR when inviting community discussions.

Reply:
Agree.